### DISCUSSION TITLE
changing text at row saved

### DISCUSSION THREAD

**Discussion by ehab-hassan** [Budibase/budibase#7993] [Help] [answered]
how I can change the approved text message here?

![Untitled-1](https://user-images.githubusercontent.com/10484295/192250142-be51b09e-8f6a-4635-8849-77b8c345a29a.jpg)

**Comment by kasgithub8** [comment, marked as answer]
Hey @ehab-hassan,

First of all, I don't know why you put this in Issues and labeled it as a bug. 
Second of all, there is a new action for buttons:
<img width="644" alt="image" src="https://user-images.githubusercontent.com/95284278/192450738-ebaab82c-26ab-41c5-bc56-13193fefd954.png">
Unfortunately, you can't disable the standard Pop-Up yet, which means 2 Pop-Ups will be shown, but Budibase already said this was something simple to implement so I guess it's coming soon!

**Comment by ehab-hassan** [comment]
ok thanks

**Comment by R0Wi** [comment]
With https://github.com/Budibase/budibase/pull/8667 you can now disable the default message and use the "Show notification" action to show a custom notification if you like
